documentation/bin/createDocu.groovy

```diff
@@ -21,7 +21,8 @@ class TemplateHelper {
 
             def props = parameters.get(it)
 
-            def defaultValue = isComplexDefault(props.defaultValue) ? renderComplexDefaultValue(props.defaultValue) : "`${props.defaultValue}`"
+            def defaultValue = isComplexDefault(props.defaultValue) ? renderComplexDefaultValue(props.defaultValue) :
+                                props.defaultValue ? "`${props.defaultValue}`" : ''
```

Comment:
check for `null`

Reply:
Aahhh ... you would like to have the empty string represented by <backtick><the empty string><backtick> and null as simply <the empty string>?

To my knowledge this does not work since backticks surrounding the empty string are also not rendered. But I will check the details.

In case I got you wrong, please update me.

Reply:
`false` should be displayed `null` not

Reply:
understood